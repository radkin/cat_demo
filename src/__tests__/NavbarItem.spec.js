import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import NavbarItem, { Counter, dataReducer } from '../Navbar/NavbarItem';

describe('NavbarItem', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<NavbarItem />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
