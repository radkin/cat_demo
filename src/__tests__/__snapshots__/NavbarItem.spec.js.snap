// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarItem snapshot renders 1`] = `
<li
  className="sc-AxirZ gpWHsK"
  onFocus={[Function]}
  onMouseEnter={[Function]}
>
  <button
    className="sc-AxjAm giZCrM"
  />
  <div
    className="sc-AxiKw hSUkaf"
  />
</li>
`;
